src/components/grid-aware/VideoHeader/VideoHeader.module.css

```diff
@@ -0,0 +1,138 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background:
+    linear-gradient(90deg, #000 0%, rgba(0, 0, 0, 0) 100%),
+    center / cover var(--background-image);
+  grid-column: 1 / span 3;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+    grid-template-rows: 214px auto;
+    padding-bottom: 55.5px;
+  }
+
+  .bleedBackground {
+    background-image: linear-gradient(360deg, #000 0%, rgba(0, 0, 0, 0) 100%), var(--background-image);
+    grid-row: 1;
+  }
+
+  .bleedMainContent {
+    grid-row: 1 / span 2;
+  }
+}
+
+/* Main grid */
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 150px 0;
+}
+
+.gridTextArea {
+  align-content: center;
```

Comment:
I don't think that this does anything, since this property only applies to grid/flex parents, not grid/flex children.

```suggestion
```

If you really want to use this, then move it to `.gridParent` (and also get rid of the `align-self` on `.gridPlayButtonArea`, since it would then be redundant).

Reply:
Okay